app/assets/javascripts/student_profile_v2/student_profile_header.js

```diff
@@ -40,16 +41,16 @@
         dom.a({
           href: Routes.school(student.school_id),
           style: styles.titleItem
-        }, student.school_name),
+        }, '\n\u2022 ' + student.school_name + '\n\u2022'),
```

Comment:
By including the bullet in the body of the `A` tag, it means the bullet itself is clickable, which probably isn't what someone would expect.  It'd be better to split it out to its own `dom.span` tag, which will also let you apply any padding if you need to.